.nf-core.yml

```diff
@@ -22,6 +22,7 @@ lint:
     - .github/PULL_REQUEST_TEMPLATE.md
     - .github/workflows/branch.yml
     - .github/workflows/linting.yml
+    - .github/workflows/linting_comment.yml
```

Comment:
What is the error you get with `linting_comment.yml`? Instead of ignoring if this file is unchanged it would make more sense to me to update it to the latest from nf-core (I think `nf-core lint --fix` would update it).

Reply:
Its an error from a depreciated actions so I thought the best method was to update that and then ignore that the test was changed

> Error: This request has been automatically failed because it uses a deprecated version of `actions/upload-artifact: v3`. Learn more: https://github.blog/changelog/2024-04-16-deprecation-notice-v3-of-the-artifact-actions/

https://github.com/phac-nml/fetchdatairidanext/actions/runs/13119779726/job/36602781549#step:1:36

Reply:
Thanks. You should be able to keep the file part of nf-core linting since when it runs in GitHub it always uses the latest `nf-core` version. Also, this is the docs for `nf-core lint --fix` for automatically fixing issues (won't fix everything though, but should automatically update github actions to latest versions from nf-core): https://nf-co.re/docs/nf-core-tools/pipelines/lint#automatically-fix-errors